skia/BUILD.gn

```diff
@@ -0,0 +1,35 @@
+#ReactNativeLocalize
+
+import("//react-native/common.gni")
+
+config("LibRNLocalizeModule_config") {
+  cflags_cc = [ "-std=c++17"]
```

Comment:
General comment, we may consider on our further implementation if valid & possible
 As these CC flags is used across the module, we can have common configuration and include it here
 Just like the below one in this file
 public_configs += [ "//build/secondary/folly:folly_config" ]